Terminal.Gui/Core/Event.cs

```diff
@@ -313,57 +348,83 @@ public class KeyEvent {
 		/// Gets a value indicating whether the Shift key was pressed.
 		/// </summary>
 		/// <value><c>true</c> if is shift; otherwise, <c>false</c>.</value>
-		public bool IsShift => (Key & Key.ShiftMask) != 0;
+		public bool IsShift => KeyModifiers.Shift;
 
 		/// <summary>
 		/// Gets a value indicating whether the Alt key was pressed (real or synthesized)
 		/// </summary>
 		/// <value><c>true</c> if is alternate; otherwise, <c>false</c>.</value>
-		public bool IsAlt => (Key & Key.AltMask) != 0;
+		public bool IsAlt => KeyModifiers.Alt;
 
 		/// <summary>
 		/// Determines whether the value is a control key (and NOT just the ctrl key)
```

Comment:
I'm not sure this comment is valid anymore.

Reply:
The information is already available in both. Which do you prefer?

Reply:
@tig I already know what to do here. I will keep both comparisons for compatibility and completely remove the modifier keys in the `Key `property of the `KeyEvent `class on `WindowsDriver`.